Lib/traceback.py

```diff
@@ -455,7 +455,7 @@ def _extract_from_extended_frame_gen(klass, frame_gen, *, limit=None,
         # information is not available.
         builtin_limit = limit is BUILTIN_EXCEPTION_LIMIT
         if limit is None or builtin_limit:
-            limit = getattr(sys, 'tracebacklimit', None)
+            limit = getattr(sys, 'tracebacklimit', 1_000)
```

Comment:
Please write a PR just for this change. As I wrote, it needs tests.

Reply:
I'm sorry, I seem to have misunderstood you. Please forgive me :(